src/server/views/dashboard/lists.njk

```diff
@@ -14,7 +14,22 @@
         </div>
       {% endif %}
     </div>
-    <table class="govuk-table dashboard-table">
+
+    <div class="govuk-grid-row govuk-!-margin-top-2">
+      {% for dashboardBox in dashboardBoxes %}
+        <div class="govuk-grid-column-one-third govuk-!-padding-top-3 dashboard-box--{{dashboardBox.cssClass}}">
+          <strong class="govuk-body govuk-!-font-weight-bold">{{ _.startCase(dashboardBox.name)}}</strong>
+          {% if dashboardBox.cssClass === "red" %}
+            <p class="govuk-body">
+              <a href="?{{dashboardBox.name}}=asc" class="govuk-body">{{dashboardBox.text}}</a>
```

Comment:
would `queryParameter` be more clear? that way it can be separate from `name`. e.g. what happens when the name has a space in it?